components/organisms/InsightPage/InsightPage.tsx

```diff
@@ -1,4 +1,4 @@
-import { useEffect, useState } from "react";
+import { useEffect, useRef, useState } from "react";
```

Comment:
nit: remove unused imports
```suggestion
import { useEffect, useState } from "react";
```